Freebox/Freebox.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <LangVersion>8.0</LangVersion>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

Comment:
Should not be needed to be consumed by a .NET 5.0 Project since .NET 5 implements all the APIs required by .NET Standard 2.0.

In fact those targets should allow most .NET Projects to consume the lib.

Unless you have a specific use case where the update is needed ?

Reply:
.NET 5 is actually replacing .NET Standard. In the case of .NET 5, unless needed for backward compatibility, there should be no references to .NET Standard.
https://devblogs.microsoft.com/dotnet/the-future-of-net-standard/

Reply:
Quoting the page you linked : "Use netstandard2.0 to share code between .NET Framework and all other platforms."

So either we have two targets : .NET Framework 4.6.x & .NET 5
Or we have only one, which will support both : .NET Standard 2.0 (because 2.1 is not interoperable with .NET Framework)

Since there is no need at the moment in this library for any of the .NET 5.0 APIs, I'd be more willing to go with the second point to clean the output up.